packages/api/src/beacon/routes/validator.ts

```diff
@@ -795,3 +795,7 @@ export function getReturnTypes(): ReturnTypes<Api> {
 function parseBuilderBoostFactor(builderBoostFactorInput?: string | number | bigint): bigint | undefined {
   return builderBoostFactorInput !== undefined ? BigInt(builderBoostFactorInput) : undefined;
 }
+
+function parseSkipRandaoVerification(skipRandaoVerification?: string): boolean {
+  return skipRandaoVerification !== undefined && skipRandaoVerification === "";
```

Comment:
> This query parameter is a flag and does not take a value.

I think checking the value doesn't even make sense as the spec states it does not take value, might be best to just checking if the flag exists

Reply:
My understanding of `does not take a value` implies an empty string in terms a query string parameter for a url. So I believe that's correct to check it.

Reply:
From this Lighthouse implementation and Nimbus it's also clear they check for the empty string value. 

https://github.com/sigp/lighthouse/blob/2a3c709f8ceec4d47c1934446511660c429fe90a/common/eth2/src/types.rs#L751

https://github.com/status-im/nimbus-eth2/blob/fc9c72f0ebc2013858f6b28c1940838c97a2f814/beacon_chain/rpc/rest_validator_api.nim#L590

Reply:
> implies an empty string in terms a query string parameter for a url

Hmm yeah might be, or potentially the difference between setting `?skip_randao_verification` and `?skip_randao_verification=`. If other clients are strictly checking it that way we should be fine as well.

Reply:
`?skip_randao_verification` is not a right query string parameter format, it query parameters always suffix with `=`

Reply:
Was looking through the RFC but didn't find anything on it, but even the openapi docs have this as an example

![image](https://github.com/ChainSafe/lodestar/assets/38436224/9d01164c-db34-4740-972a-a9bfe4d6d474)


I would expect that `?skip_randao_verification` is allowed by fastify but in the end it might be an implementation detail.